TestCases.md

```diff
@@ -0,0 +1,487 @@
+12. Verify that the seed phrase matches the one generated during wallet creation.
+13. Verify that the "Copy" button is visible and functional.
+14. Ensure that clicking the "Copy" button copies the entire mnemonic phrase to the clipboard.
+15. Verify that the "OK" button is visible and functional.
+16. Ensure that clicking the "OK" button or the close (cross) button in the upper-right corner closes the modal.
+17. Ensure that once the modal is closed, the mnemonics are no longer accessible unless the process is repeated (with a password prompt if not within the 5-minute period).
+18. Verify behavior when the user repeatedly enters an incorrect password when trying to back up mnemonics.
+
+## Export Wallet File
+
+1. Verify that the "Export Wallet File" button is visible in the settings menu.
+2. Ensure the button is clearly labeled and accessible.
+3. Verify that clicking the "Export Wallet File" button immediately triggers the download of the wallet JSON file.
+4. Ensure the file is saved in the user’s default download directory.
+5. Confirm that the downloaded wallet file is in the correct JSON format.
+6. Verify that the file contains the expected wallet data and is not corrupted.
+7. Ensure that if the wallet file export fails for any reason (e.g., disk space, permission issues), an appropriate error message is shown, and the user is prompted to try again.
+
+## Wipe Out
+
+1. Verify that the "Wipe Out" button is visible in the settings menu.
+2. Ensure the button is clearly labeled and has a distinct color.
+3. Verify that clicking the "Wipe Out" button triggers a modal.
+4. Confirm that the modal contains two buttons: "Cancel" and "Wipe Out."
+5. Verify that clicking the "Cancel" button closes the modal without performing any action.
+6. Ensure the wallet data remains intact after canceling.
+7. Verify that clicking the "Wipe Out" button within the modal deletes all wallet data.
+8. Ensure the user is logged out and redirected to the initial setup or login page.
+9. Confirm that there is no way to recover the wallet after confirming the wipeout
+
+# Edge Cases
+
+1. Verify that reloading the mnemonic phrase page generates a new mnemonic phrase.
+2. Ensure the user is prompted to start the process again if the page is reloaded on the further steps.
+3. Ensure that user confirmation is required if the page is reloaded (to prevent accidental data loss).
+4. Check behavior when the session expires while on the mnemonic phrase setup steps.
+5. Check behavior when the session expires while on the password setup page.
+6. Ensure the user is redirected to the login/main page appropriately.
+7. Verify behavior if clipboard permissions are denied.
+8. Ensure an appropriate error message is displayed if clipboard access is not available.
+9. Verify behavior when the network connection is lost during the tested process.
+10. Check behavior under slow network conditions.
```

Comment:
Especially places where a response from the BE is expected. Last case I got is Fee calculation during transaction verification.